app/src/main/java/ch/epfl/sdp/blindwar/database/ImageDatabase.kt

```diff
@@ -53,31 +30,20 @@ object ImageDatabase {
                     Snackbar.make(view, "Image uploaded", Snackbar.LENGTH_LONG).show()
 //                    UserDatabase.addProfilePicture(user!!.uid, uploadedImageRef.path)
                 }
-            /*
+        /*
         .addOnFailureListener {
             Toast.makeText(getApplicationContext(), "Failed to upload file",
                 Toast.LENGTH_LONG).show()
-        }*/
+        } */
         } else {
             uploadedImageRef.putFile(imageURI)
         }
         return uploadedImageRef.path
     }
 
-
-    fun downloadProfilePicture(imagePath: String, imageView: ImageView, context: Context): String {

```

Comment:
Why was this function deleted ? I don't see it somewhere else.

Reply:
The profile picture is fetched (using Glide) in the presentation layer of the app (ProfileFragment/PlayMenuFragment) to avoid passing a reference of a context and view to a database function.